tests/components/snmp/test_integer_switch.py

```diff
@@ -0,0 +1,62 @@
+"""SNMP switch tests."""
```

Comment:
Please rename to test_switch.py

Reply:
done in 5880c6f3fd

(It was named test_integer_switch to match `test_*_sensor` , with a view to adding test_string_switch in #122944. I assume all switch tests should be contained in a single test file?)

Reply:
Yes!